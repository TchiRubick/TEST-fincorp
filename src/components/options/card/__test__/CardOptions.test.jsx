import React from 'react';
import ReactDom from 'react-dom';
import { cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import CardOptions from '../CardOptions';
import img from '../../../../assets/img/logo.png';

afterEach(cleanup);

it('renders good', () => {
  const div = document.createElement('div');

  ReactDom.render(<CardOptions title="test" color="green" image={img}>test</CardOptions>, div);
});

it('as expected snapshot 1', () => {
  const tree = renderer.create(<CardOptions title="test" color="green" image={img}>test</CardOptions>).toJSON();
  expect(tree).toMatchSnapshot();
});
