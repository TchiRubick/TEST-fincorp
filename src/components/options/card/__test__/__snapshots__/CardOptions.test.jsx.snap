// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`as expected snapshot 1 1`] = `
<div
  className="comp-card-options"
>
  <div
    className="comp-card-options__image"
  >
    <img
      alt="test_image"
      src="logo.png"
      style={
        Object {
          "backgroundColor": "green",
          "height": "6rem",
          "width": "6rem",
        }
      }
    />
  </div>
  <div
    className="comp-card-options__title"
  >
    <span
      className="comp-card-options__title-text"
    >
      test
    </span>
  </div>
  <div
    className="comp-card-options__text"
  >
    test
  </div>
</div>
`;
